libraries/AP_HAL_ChibiOS/hwdef/DAKEFPVF405/defaults.parm

```diff
@@ -0,0 +1,11 @@
+# setup SERIAL2 for RCIN
+SERIAL2_PROTOCOL 23
+
+# disable GPS on Serial 4
+SERIAL4_PROTOCOL -1
+
+# setup SERIAL1 for GPS
+SERIAL1_PROTOCOL 5
+
+# Onboard ELRS
+RC2_REVERSED 1 # Prevent inverse pitch controls
```

Comment:
this is discouraged since some TX normal output is reversed, although infrequent

Reply:
you should set OSD_TYPE2 5 for DisplayPort in defaults
and RELAY_EN and the functions for RELAY2 and RELAY since just defining the pins hwdef no longer means they are activated....and not sure what GPIO70 does (RELAY2) since its also labeled as a camera switch in the hwdef...maybe its the 12V video supply switch??

Reply:
Corrected as required. RELAY2 use VTX supply switch.

Reply:
you did not add OSD_TYPE2 5 to defaults.parm